### PR TITLE
Expose rpc service for remote backup and restore

### PR DESCRIPTION
**Pull request by naseemkullah**
Signed-off-by: Naseem Ullah <naseemkullah@gmail.com>

### CODE REVIEW

**Comment by naseemkullah** [comment]
Hi @jackzampolin, can you please take a look?

File: influxdb/templates/service.yaml [hunk 1]
```diff
@@ -35,5 +35,10 @@ spec:
     port: {{ .Values.config.opentsdb.bind_address }}
     targetPort: {{ .Values.config.opentsdb.bind_address }}
   {{- end }}
+  {{- if .Values.exposeRpcService }}
```
Comment (jackzampolin) [inline influxdb/templates/service.yaml]: Maybe move the `bind_address` under a `rpc`

```yaml
config:
  rpc:
    enabled: true
    bind_address: 8088
```

File: influxdb/values.yaml [hunk 2]
```diff
@@ -159,3 +159,5 @@ config:
     log_enabled: true
     enabled: true
     run_interval: 1s
+
+exposeRpcService: true
```
Comment (jackzampolin) [inline influxdb/values.yaml]: See above comment.

**Comment by jackzampolin** [comment]
utACK, looks great! cc @timhallinflux

**Pull request merged**